packages/core/src/card/card.element.ts

```diff
@@ -0,0 +1,62 @@
+import { setAttributes } from '@cds/core/internal';
+import { CdsInternalPanel } from '@cds/core/internal-components/panel';
+import { property } from 'lit-element';
+import { styles } from './card.element.css.js';
+
+export const CdsCardTagName = 'cds-card';
+
+/**
+ * Web component card.
+ *
+ * ```typescript
+ * import '@cds/core/card/register.js';
+ * ```
+ *
+ * ```html
+ * <cds-card>
+     <div cds-layout="vertical">
+       <div class="header" cds-layout="p:lg">
+         <p cds-text="h4">Card Title</p>
+       </div>
+
+       <cds-divider></cds-divider>
+
+       <div class="body" cds-layout="p:lg" cds-text="p1" style="overflow:auto">
+         ${longerContent}
+       </div>
+
+       <cds-divider></cds-divider>
+
+       <div class="footer" cds-layout="p:lg">
+         <cds-button action="flat"> View </cds-button>
+       </div>
+     </div>
+ * </cds-card>
+ * ```
+ *
+ * @element cds-card
+ * @slot - For projecting card content
+ * @cssprop --width
+ * @cssprop --height
+ * @cssprop --color
+ * @cssprop --overflow
+ * @cssprop --overflow-x
+ * @cssprop --overflow-y
+ * @cssprop --background
+ * @cssprop --border
+ * @cssprop --border-radius
+ * @cssprop --padding
+ * @cssprop --box-shadow
+ */
+export class CdsCard extends CdsInternalPanel {
+  @property({ type: String }) role = 'region';
+
+  connectedCallback() {
+    super.connectedCallback();
+    setAttributes(this, ['role', this.role]);
+  }
```

Comment:
You can remove this block by adding reflect true to the prop decorator, then it will add the attr for you automatically.

```typescript
@property({ type: String, reflect: true }) role = 'region';
```